tests/queries/0_stateless/03271_date_to_datetime_saturation.sql

```diff
@@ -0,0 +1,105 @@
+drop table if exists test;
+
+create table test (stamp Date) engine MergeTree order by stamp;
+
+insert into test select '2024-10-30' from numbers(100);
+insert into test select '2024-11-19' from numbers(100);
+insert into test select '2149-06-06' from numbers(100);
+
+optimize table test final;
+
+-- implicit toDateTime (always saturate)
+select count() from test where stamp >= parseDateTimeBestEffort('2024-11-01');
```

Comment:
Could you please enable `echoOn` for this file? Otherwise matching the query with the reference is quite tiresome